raiden/app.py

```diff
@@ -95,12 +95,21 @@ def __init__(
 
         # Check if files with older versions of the DB exist, emit a warning
         db_base_path = os.path.dirname(config['database_path'])
-        if older_db_files_exist(db_base_path):
-            log.warning(
-                'Older versions of the database exist in '
-                f'{db_base_path}. Since a newer breaking version is introduced, '
-                'it is advised that you leave all token networks before upgrading and '
-                'then proceed with the upgrade.',
+        old_version_path = older_db_files_exist(db_base_path)
+        if old_version_path:
+            old_version_file = os.path.basename(old_version_path)
```

Comment:
Wouldn't it be better to return all the old files from `older_db_files_exist()` so that if the user has multiple old databases, he does not need to follow what the error below tells him multiple times?

Reply:
A normal user should have this error happening only once, because each breaking upgrade should complain. We may have multiple db versions though